app/auth/callback/unilogin/route.ts

```diff
@@ -83,29 +101,14 @@ export async function GET(request: NextRequest) {
     // await session.save()
```

Comment:
Is this commented out code necessary here?

Reply:
That change I made too fast. Thx for catching that!